plugins/woocommerce-admin/client/inbox-panel/test/index.js

```diff
@@ -101,50 +100,89 @@ describe( 'getUnreadNotesCount', () => {
 	const lastRead = 1589285995243;
 
 	test( 'should return 4, 1 of the notes was read', () => {
-		const unreadCount = getUnreadNotesCount( notes, lastRead );
+		const unreadCount = getUnreadNotesCount( NOTES, lastRead );
 		expect( unreadCount ).toEqual( 4 );
 	} );
 
 	test( 'should return 3, 1 of the notes was read and 1 is deleted', () => {
-		notes[ 3 ].is_deleted = true;
+		const notes = [
+			{ ...NOTES[ 0 ] },
+			{ ...NOTES[ 1 ] },
+			{ ...NOTES[ 2 ] },
+			{ ...NOTES[ 3 ], is_deleted: true },
+			{ ...NOTES[ 4 ] },
+		];
```

Comment:
This is legit, but you could consider creating having a function such as the following to create a deep copy...

```js
function deepCopy( sourceArray ) {
    return JSON.parse( JSON.stringify( sourceArray ) );
}
```

And then just make the modification on the copy.

I'm fine with the code staying as is. But, if you want to refactor to create the `deepCopy()` function, just ping me for a re-review.